CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+
+# Contribution Rules📚:
+
+- You are allowed to make pull requests. We just merge it ;)
+- Do NOT remove other content.
+- Styling/code can be pretty, ugly or stupid, big or small as long as it works
+- Try to keep pull requests small to minimize merge conflicts
+
+
```

Comment:
I think we need to include that to work on projects, Flutter is needed. (Not exactly at this place but somewhere).